src/main/java/spring/info/config/WebConfig.java

```diff
@@ -0,0 +1,26 @@
+package spring.info.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+/**
+ * @author Sergey Klunniy
+ */
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages ={"spring.info.controllers"})
+public class WebConfig {
+
+    @Bean
+    public InternalResourceViewResolver resolver() {
```

Comment:
```suggestion
    public InternalResourceViewResolver getResolver() {
```
назви методів - дієслово або фраза на його основі (get, put, createNewFile)

Reply:
виправив